.example.env

```diff
@@ -0,0 +1,18 @@
+## Example environment file for docker-compose and builders
+
+COMPOSE_PROJECT_NAME=postgis
+
+## For build arguments
+DISTRO=debian
+IMAGE_VERSION=bullseye
+IMAGE_VARIANT=slim
+# Set GENERATE_ALL_LOCALE to empty value or 0 to build just default LOCALE: en_US.UTF-8
+GENERATE_ALL_LOCALE=1
+# Set the language if you need to specify LANG locale at build time
+LANG=en_US.UTF-8
+# locale filter to include in the locale generator
+LANGS="en_US.UTF-8,id_ID.UTF-8"
```

Comment:
I added this build args to only build default locale.
Useful for creating slim base images to do the scenario tests (less time to build).
The reason why we use both en_US and id_ID is because it's used in the tests. So this is the default.

Both `LANG` and `LANGS` only used if `GENERATE_ALL_LOCALE=0` or empty (not generating all locale).